pages/styles/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+    background-color: #FEBCAE;
+    color:#4120A9;
+}
+
+#index #about {
```

Comment:
These styles aren't being applied because the rule generated by space-separated selectors here is "Select any element with an ID of `about` that is a child of any element with an ID of `index`". I think you meant to use a comma to separate them, i.e.
```css
#index, #about {
  ...
}
```